Workbooks/Azure Monitor - Workspaces/Metrics Usage Insights/Metrics Usage Insights.workbook

```diff
@@ -119,7 +128,7 @@
           {
             "type": 1,
             "content": {
-              "json": "Please verify that your Azure Monitor workspace is actively ingesting data and ensure that metrics usage insights are available in the region where your AMW was created. [Find the list of supported regions here](https://learn.microsoft.com/azure/azure-monitor/metrics/metrics-usage-insights#supported-regions)",
+              "json": "Please verify that metrics usage insights is available in the region where your AMW was created. [Find the list of supported regions here](https://review.learn.microsoft.com/azure/azure-monitor/essentials/metrics-usage-insights?branch=pr-en-us-981&tabs=portal#supported-regions)",
```

Comment:
this switches to an internal review doc that woudl only be available to microsoft employess/etc.  intentional?